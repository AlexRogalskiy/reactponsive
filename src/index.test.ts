import * as RP from '.';

describe('index file', () => {
  test('snapshot', () => {
    expect(RP).toMatchSnapshot();
  });
});
