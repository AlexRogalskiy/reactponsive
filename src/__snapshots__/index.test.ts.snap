// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index file snapshot 1`] = `
Object {
  "AliasProvider": [Function],
  "ReactPonsive": [Function],
  "ReactPonsiveToggle": [Function],
  "ReactPonsiveValue": [Function],
  "useReactPonsive": [Function],
  "useReactPonsiveToggle": [Function],
  "useReactPonsiveValue": [Function],
  "withReactPonsive": [Function],
  "withReactPonsiveToggle": [Function],
}
`;
